CHANGELOG.md

```diff
@@ -10,6 +10,103 @@
 
 - "You miss 100 percent of the chances you don't take. — Wayne Gretzky" — Michael Scott
 
+## 8.35.0
+
+### Beta release of the official Nuxt Sentry SDK
+
+This release marks the beta release of the `@sentry/nuxt` Sentry SDK. For details on how to use it, check out the
+[Sentry Nuxt SDK README](https://github.com/getsentry/sentry-javascript/tree/develop/packages/nuxt). Please reach out on
+[GitHub](https://github.com/getsentry/sentry-javascript/issues/new/choose) if you have any feedback or concerns.
+
+- **feat(nuxt): Make dynamic import() wrapping default
+  ([#13958](https://github.com/getsentry/sentry-javascript/pull/13958))** (BREAKING)
+- **feat(nuxt): Add Rollup plugin to wrap server entry with `import()`
+  ([#13945](https://github.com/getsentry/sentry-javascript/pull/13945))**
+
+There is no need to add the Node `--import` flag anymore because the SDK now wraps your application with a dynamic
+`import()`. Please update your start command to avoid initializing Sentry twice (BREAKING CHANGE). If you prefer to use
+the `--import` flag, you can enable this option in your `nuxt.config.ts`:
+
+```js
+sentry: {
+  dynamicImportForServerEntry: false;
+}
+```
+
+- **feat(nuxt): Respect user-provided source map generation settings
+  ([#14020](https://github.com/getsentry/sentry-javascript/pull/14020))**
+
+As Nuxt sets `sourcemap.client: false`, it's not possible to determine whether this setting was set by you or the
+framework. Set this manually like this:
+
+```js
+export default defineNuxtConfig({
+  sourcemap: {
+    client: true,
+  },
+});
+```
```

Comment:
```suggestion
We now require you to explicitly enable sourcemaps for the clientside so that Sentry can unminify your erros. We made this change so source maps aren't accidentally leaked to the public. You can do so as follows:

```js
export default defineNuxtConfig({
  sourcemap: {
    client: true,
  },
});
```
```

Reply:
What do you think of this: 

We now respect your source map settings if they are defined, so source maps aren't accidentally leaked to the public. Since Nuxt defaults `sourcemap.client` to `false`, we can't tell if this was set by you or the framework. We now require you to explicitly enable sourcemaps for client-side source maps for Sentry can un-minify your errors:

Reply:
Assume you don't know what sourcemaps are, or what nuxt is doing wrt sourcemaps. This is the audience we should write the changelog entry for.

Reply:
From this standpoint, your suggestion makes sense. I'll change that 👍